tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -92,20 +105,25 @@ function HeaderComp() {
                   )}
                 </li>
                 <li className="usa-nav__secondary-item">
-                  <a
-                    className="sign-out-link"
-                    href={
-                      user && user.email
-                        ? `${process.env.REACT_APP_BACKEND_URL}/logout/oidc`
-                        : `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
+                  {(() => {
```

Comment:
Nit but can this IIFE be changed to a ternary expression for readability?

Reply:
Ah, yes, I can do that. When I originally wrote it there were gonna be extra variables in that context.